ycms/cms/views/patients/patients_list_view.py

```diff
@@ -27,4 +30,79 @@ def get_context_data(self, **kwargs):
         :return: Response for filtered offers
         :rtype: ~django.template.response.TemplateResponse
         """
-        return {"patients": Patient.objects.all(), **super().get_context_data(**kwargs)}
+        return {
+            "patients": Patient.objects.all().order_by("-created_at"),
+            **super().get_context_data(**kwargs),
+        }
+
+    def post(self, request, *args, **kwargs):
+        r"""
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: Redirect to list of patients
+        :rtype: ~django.http.HttpResponseRedirect
+        """
+
+        if len(request.POST) == 2:
+            patient = Patient.objects.get(id=request.POST["id"])
+
+            messages.success(
+                request,
+                _('Patient "{} {}" has been deleted.').format(
+                    patient.first_name, patient.last_name
+                ),
+            )
+
+            patient.delete()
```

Comment:
```suggestion
            patient.delete()
            
            messages.success(
                request,
                _('Patient "{} {}" has been deleted.').format(
                    patient.first_name, patient.last_name
                ),
            )
```
veeery nitpicky, but technically... :D